meson.build

```
@@ -976,7 +976,10 @@ wayland = {
              dependency('wayland-cursor', version: '>= 1.15.0', required: get_option('wayland')),
              dependency('wayland-protocols', version: '>= 1.15', required: get_option('wayland')),
              dependency('xkbcommon', version: '>= 0.3.0', required: get_option('wayland'))],
-    'header': cc.has_header('linux/input-event-codes.h', required: get_option('wayland')),
+    'header': cc.has_header('linux/input-event-codes.h', required: get_option('wayland'),
+                            # Pass CFLAGS from a related package as a hint for non-Linux
```

Sorry to bug you here again but could you change this to something like the following?
```suggestion
                            # BSDs may need to link to libwayland in order to find the linux header
```

---

It's less accurate:
- `has_header` doesn't link anything
- the header isn't part of libwayland package
- NetBSD and OpenBSD may take a different approach
- the commit message properly explains "why?"

---

How about `# Workaround so some BSDs can find the linux header`?